assets/css/home.css

```
@@ -0,0 +1,2222 @@
+}
+
+.home-page__hero {
+  --hero-space-sm: clamp(0.75rem, 1.5vh, 1rem);
+  --hero-space-md: clamp(1.15rem, 2.2vh, 1.65rem);
+  --hero-space-lg: clamp(2rem, 4.5vh, 3.25rem);
+  --hero-col-gap: clamp(2rem, 5vw, 4.5rem);
+  --hero-diagram-max: clamp(20rem, 44vw, 40rem);
+  --hero-diagram-min-h: clamp(11rem, 24vw, 16rem);
+  --hero-diagram-max-h: min(48vh, 24rem);
+  --hero-diagram-inset-right: clamp(1.25rem, 3.5vw, 2.75rem);
+  margin: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)